src/Component/BlazorComponent/Components/Treeview/BTreeview.razor.cs

```diff
@@ -292,9 +294,9 @@ public async Task EmitSelectedAsync()
         {
             var selected = Nodes.Values.Where(r =>
             {
-                if (SelectionType == SelectionType.Independent)
+                if (SelectionType == SelectionType.Independent || SelectionType == SelectionType.Standalone)
```

Comment:
I think `LeafButIndependentParent` is better than `Standalone`.

```suggestion
                if (SelectionType is SelectionType.Independent or SelectionType.LeafButIndependentParent)
```